### PR TITLE
updated build.gradle with new URLs, removed wireless charging local chat spam

### PR DESCRIPTION
**Pull request by bombcar**
replaced spaces with tabs
added some files to the gitignore

### CODE REVIEW

**Comment by bombcar** [comment]
Currently as a draft - I can't get gt++ to build without these changes but @draknyte1 says they shouldn't be necessary.

I blame mojang

**Comment by bombcar** [comment]
Can someone tell me if the build changes are correct/needed? Perhaps it would build without them now.

File: build.gradle [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 		maven {
-			name = "gt"
-			url = "https://gregtech.overminddl1.com/"
-		}	
-        maven {
-            name = "sonatype"
-            url = "https://oss.sonatype.org/content/repositories/snapshots/"
-        }
+			name = "jitpack"
+			url = "https://jitpack.io"
```
Comment (glowredman) [inline build.gradle]: What do you need Jitpack for? Forge is pulled from maven.minecraftforge.net (which you defined below)

Comment (bombcar) [reply build.gradle]: I don't actually know, I was just copying things in until it built - but looks like that wasn't needed.

Comment (bombcar) [reply build.gradle]: hmm how do I indicate that 1142e97 below resolves this?

**Review by GlodBlock — APPROVED** [review]

**Pull request merged**